<launch>

	<!--USV Model-->
    <node name="dynamic_model_simulate" type="dynamic_model_simulate" pkg="usv_control" />

	<!-- Guidance Law -->
    <node pkg="usv_control" type="los.py" name="los" />

    <!--Controler-->
    <node name="asmc" pkg="usv_control" type="asmc" >
	<param name = "k_u" value = "0.1" />
	<param name = "k_psi" value = "0.2" />
	<param name = "kmin_u" value = "0.05" />
	<param name = "kmin_psi" value = "0.2" />
	<param name = "k2_u" value = "0.02" />
	<param name = "k2_psi" value = "0.1" />
	<param name = "mu_u" value = "0.05" />
	<param name = "mu_psi" value = "0.1" />
	<param name = "lambda_u" value = "0.001" />
	<param name = "lambda_psi" value = "1" />
    </node>

</launch>